types/fhir/index.d.ts

```diff
@@ -17761,8 +17762,8 @@ declare module fhir {
          * This patient is known to be an animal (non-human)
          */
         animal?: PatientAnimal;
-        /**
-         * A list of Languages which may be used to communicate with the patient about his or her health
+        /**their
```

Comment:
oops, this was added by mistake i suppose ;)

Reply:
Oops! Nice catch! 🙈